components/eventing-controller/utils/utils.go

```diff
@@ -95,3 +100,10 @@ func GetSinkData(sink string) (string, []string, error) {
 	subDomains := strings.Split(trimmedHost, ".")
 	return trimmedHost, subDomains, nil
 }
+
+func GetCloudEvent() event.Event {
+	newEvent := cloudevents.NewEvent()
+	newEvent.SetType(GetRandString(randStringlength))
+	newEvent.SetID(GetRandString(randStringlength))
+	return newEvent
+}
```

Comment:
(nit) Its setting a random type in CE, maybe mention it in name e.g. `GetCloudEventWithRandType()`?

Reply:
Agreed to provide actual type from the subscription instead of a random one